src/main/java/com/extendedclip/papi/bungeeexpansion/BungeeExpansion.java

```diff
@@ -88,12 +96,16 @@ public String onRequest(final OfflinePlayer player, String identifier) {
 
     @Override
     public void start() {
+        if (hasString("online"))
+            this.online = getString("online", "Online");
+        if (hasString("offline"))
+            this.offline = getString("offline", "Offline");
```

Comment:
Instead of making this extra method, just update the `getDefaults()` method to include these new entries and it should update accordingly, especially since the getString method returns the default given if returned path is null.

Reply:
made